spring-data-vmware-gemfire/build.gradle.kts

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) VMware, Inc. 2022. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+import com.google.cloud.storage.BlobId
+import com.google.cloud.storage.BlobInfo
+import com.google.cloud.storage.StorageOptions
+import org.apache.tools.ant.taskdefs.condition.Os
+
+
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath(libs.google.cloud.storage)
+  }
+}
+
+plugins {
+  id("java-library")
+  id("gemfire-repo-artifact-publishing")
+  alias(libs.plugins.lombok)
+  alias(libs.plugins.dependency.management)
+}
+
+java {
+  withJavadocJar()
+  withSourcesJar()
+
+  toolchain{ languageVersion.set(JavaLanguageVersion.of(8))}
+}
+
+tasks.named<Javadoc>("javadoc") {
+  title = "Spring Data for VMware GemFire Java API Reference"
```

Comment:
Why do we not have a data version of 2.6 in the title?

Reply:
I added the version in the title